### ISSUE TITLE
Download/update on open

### ISSUE THREAD

**Issue by dmaglio** [yongkangchen/remote-sync#85]
If I open a folder with sftp/ftp configuration, I wish they would update from webserver

**Comment by yongkangchen** [comment]
I can not understand. You mean it should auto update from webserver when atom open a folder?

**Comment by dmaglio** [comment]
yes, check difference and update local folder from remote

**Comment by i-c** [comment]
Instead of folders, this seems like it would work better for individual files, like how [Sublime SFTP](http://wbond.net/sublime_packages/sftp) does it. If 'download on open' is enabled, it checks if the file on the server has a newer modification date than the local file, and if it does downloads it.

**Comment by jbh** [comment]
Yes, +1 on the Sublime SFTP reference. This is the main thing keeping me from using Atom at work, which I would love to do.

**Comment by stilliard** [comment]
+1, This would be really useful.

**Comment by panesarg** [comment]
I would like to put in a request for this as well. It's the only thing that's stopping me from switching to Atom from Sublime.

**Comment by leona1das** [comment]
+1, it's a pain if you switch between branches on the server and you forget to download before opening files. Yes, I can sync full project but it's a really nice feature to just have this option :)

**Comment by jirihofman** [comment]
+1, I would really like that feature as well
